platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -977,7 +977,7 @@ private boolean obbIsCorrupted(String f, String main_pack_md5) {
 
 	public boolean gotTouchEvent(final MotionEvent event) {
 
-		final int evcount = event.getPointerCount();
+		int evcount = event.getPointerCount();
```

Comment:
I noticed most `final`s where there so that the `Runnable` classes could access them. That's a requirement in Java for a class to access the scope of the function it's declared in.

I started removing them, but eventually I thought that they could very well stay anyway. But I forgot to restore this one.